src/main/java/ladysnake/satin/mixin/client/event/GameRendererMixin.java

```diff
@@ -55,14 +55,6 @@ private void hookShaderRender(float tickDelta, long nanoTime, boolean renderLeve
         ShaderEffectRenderCallback.EVENT.invoker().renderShaderEffects(tickDelta);
     }
 
-    @Inject(
-            at = @At(value = "INVOKE", target = "Lnet/minecraft/client/MinecraftClient;getFramebuffer()Lnet/minecraft/client/gl/Framebuffer;"),
-            method = "render"
-    )
-    private void fixMojankGl(float tickDelta, long nanoTime, boolean renderLevel, CallbackInfo info) {
-        RenderSystem.enableTexture();
-    }
-
```

Comment:
Dis Mojang add a similar call ?

Reply:
`RenderSystem.enableTexture()` was removed. All the features of the TestMod are working fine